buildscripts/azure/azure-linux-macos.yml

```diff
@@ -13,6 +13,7 @@ jobs:
 
   steps:
     - script: |
+        set -e
```

Comment:
Would it be worth adding `set -e` to the start of all of these `script` steps to catch similar issues in future?

Reply:
Yes, I hadn't noticed that it was on every step in the llvmlite list of steps. Now added.